src/map/SidebarList.tsx

```diff
@@ -0,0 +1,70 @@
+// DKDK an example of a sidebar component
+import React from 'react';
+import { Sidebar, Tab } from './SidebarReactCore'
+//DKDK modified version of typescript definition for react-leaflettaken
+import { SidebarProps } from './type-react-leaflet-sidebarv2'
+
+//DKDK testing to separate a component for tab content
+import TabHomeContent from './TabHomeContent'
+
+/**
+ * DKDK this is an example of Sidebar component
+ */
+export default function SidebarList(props: SidebarProps) {
+  /**
+   * DKDK think it may be better to separate Tabs as sub-components in the future?
+   */
+  let testText = 'this is Home'
+  return (
+    <Sidebar
+        id={props.id}
```

Comment:
I think all these can be replaced with {...props}

Reply:
yes indeed in that example 👍 It is because the SidebarList.tsx is, as I mentioned earlier, a manual configuration for tabs - no props to change tabs.

However, we will have the case that we send additional props for tabs (like data or query/query terms for requesting data at tab content components - like we used Solr query for data to describe detailed table, plot, etc.)